apps/src/dance/blocks.js

```diff
@@ -103,6 +103,7 @@ const customInputTypes = {
 export default {
   customInputTypes,
   install(blockly, blockInstallOptions) {
+    blockly.fieldRegistry.register('field_dance_ai', CdoFieldDanceAi);
```

Comment:
Not sure where the right place for this is. Maybe a common repository of custom fields? Maybe a separate one-time setup function? The other place we do this is in Music Lab's [setup.ts](https://github.com/code-dot-org/code-dot-org/blob/8bb4c377b7eb9850048e8a325b2514c2e95b76ce/apps/src/music/blockly/setup.ts#L45), which is called before Music Lab is mounted on the page.

Reply:
We tend to register custom fields like this right in the wrapper, but haven't really worried about lab-specific fields so far.